src/components/ha-selector/ha-selector-location.ts

```diff
@@ -26,6 +26,7 @@ export class HaLocationSelector extends LitElement {
 
   protected render() {
     return html`
+      ${this.label ? this.label : ""}
```

Comment:
I think it was omitted on purpose, as this is kinda assumed at this point?

@matthiasdebaat WDYT?

Reply:
I agree. Why do you want to add this @joostlek?

Reply:
I found it strange that we do define this label in core, but it isn't shown. For example at Tankerkoenig, you just get a map without any context 
![image](https://github.com/home-assistant/frontend/assets/7083755/f9e4fefc-3105-445a-94f9-d2669c729a82)
(of course this would be solved with a description or something).

And maybe for integrations that require 2 locations (google travel time might be an example) it would be useful to add some context.

But personally I just noticed that this wasn't consistent so I am also fine if we just keep it as is.

Reply:
I think it makes sense to add this, the user of the form should make the decision if the label is needed or not, for example if there are multiple locations to be picked

Reply:
Fair point. It's strange that the name doesn't show up when you set one. Though in the example of Tankerkoenig, the name is required and not a decision of the user.